test/wdio.conf.ts

```diff
@@ -162,7 +162,7 @@ export const config: Options.Testrunner = {
   // Define all options that are relevant for the WebdriverIO instance here
   //
   // Level of logging verbosity: trace | debug | info | warn | error | silent
-  logLevel: EnvironmentSettings.getInstance().logLevel,
+  logLevel: 'warn',
```

Comment:
This still needs to be configurable. However, given this change, `warn` gives us an acceptable level of console output for typical use. Maybe the value here should be based on the selected log level, via env var, but adjusted to "fit" wdio's application of log levels. If log level is `info` from env, it is then mapped to `warn`, `debug` -> `info`, etc.